src/sentry/demo/demo_start.py

```diff
@@ -172,21 +174,37 @@ def get_one_release(org: Organization, project_slug: Optional[str]):
     return f"/organizations/{org.slug}/releases/{version}/?project={project.id}"
 
 
-def get_one_issue(org: Organization, project_slug: Optional[str]):
+def get_one_issue(org: Organization, project_slug: Optional[str], error_type: Optional[str]):
     group_query = Group.objects.filter(project__organization=org)
-    if project_slug:
+
+    if project_slug and error_type:
         group_query = group_query.filter(project__slug=project_slug)
-    group = group_query.first()
+
+        for group in group_query:
```

Comment:
@roggenkemper Maybe we could use `filter` to be more functional here?

Reply:
I think it would be more complicated/maybe not work since we would need to filter by what's inside metadata  and some fields might not exist